README.md

```diff
@@ -2,19 +2,6 @@
 
 Prototype Contract for Subscription Payments
 
-## k8s transcription
```

Comment:
I was literally just looking for this in the `graph-subscriptions-api` lol. Evidently did not move it over. 😆